### PR TITLE
bumping stack lts-18.11

### PR DESCRIPTION
**Pull request by erykciepiela**

### CODE REVIEW

File: ruby-marshal.cabal [hunk 1]
```diff
@@ -28,13 +28,13 @@ library
...
-    , string-conv >= 0.1    && <= 0.2
-    , mtl         >= 2.1.0  && <= 2.3.0
-    , vector      >= 0.10.0 && <  0.12.1
+      base
+    , cereal
+    , bytestring
+    , containers
+    , string-conv
+    , mtl
+    , vector
```
Comment (zyla) [inline ruby-marshal.cabal]: Let's not drop version bounds completely, instead relax them as needed

**Review by zyla — CHANGES_REQUESTED** [review]

File: ruby-marshal.cabal (hunk 1 above)
Comment (erykciepiela) [reply ruby-marshal.cabal]: done

Comment (zyla) [reply ruby-marshal.cabal]: Thanks!

File: ruby-marshal.cabal [hunk 2]
```diff
@@ -29,12 +29,12 @@ library
...
       base        >= 4.7    && <= 5
-    , cereal      >= 0.4.0  && <  0.5.6
+    , cereal      >= 0.4.0  && <  0.5.9
     , bytestring  >= 0.9.0  && <= 0.12.0
-    , containers  >= 0.5.0  && <= 0.6.0
+    , containers  >= 0.5.0  && <  0.6.6
     , string-conv >= 0.1    && <= 0.2
     , mtl         >= 2.1.0  && <= 2.3.0
-    , vector      >= 0.10.0 && <  0.12.1
+    , vector      >= 0.10.0 && <  0.12.4
```
Comment (zyla) [inline ruby-marshal.cabal:37]: I wonder why such specific bounds are used. I think `< 0.13` should suffice here. (0.13 would be the next "major" version)

**Review by zyla — APPROVED** [review]

File: ruby-marshal.cabal (hunk 2 above)
Comment (erykciepiela) [reply ruby-marshal.cabal:37]: Yes, what I find unclear is the discrepancy between https://pvp.haskell.org/ and versioning used in many hackage packages that use a.b.c.d scheme. Is b or c the major version in these cases?

**Pull request merged**